### PR TITLE
typescript code generation post processing

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[UI-Client-Library aus OpenAPI generieren](https://trello.com/c/Barx5fXr/1535-ui-client-library-aus-openapi-generieren)

File: .generation/post-process/typescript.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/bin/python3
```
Comment (michaelmattig) [inline .generation/post-process/typescript.py:1]: Do we need to override the user agent here too?

File: typescript/src/apis/SessionApi.ts [hunk 2]
```diff
@@ -139,6 +149,69 @@ export class SessionApi extends runtime.BaseAPI {
         await this.logoutHandlerRaw(initOverrides);
     }
 
+    /**
+     * Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.  # Errors  This call fails if Open ID Connect is disabled, misconfigured or the Id Provider is unreachable.
```
Comment (michaelmattig) [inline typescript/src/apis/SessionApi.ts:153]: why are all comments doubled?

Comment (ChristianBeilschmidt) [reply typescript/src/apis/SessionApi.ts:153]: We have it as our Rust-Doc and in the Utoipa-Macro, this means the generated JSON looks like this:

```json
"/oidcInit": {
      "post": {
        "tags": [
          "Session"
        ],
        "summary": "Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.",
        "description": "Initializes the Open Id Connect login procedure by requesting a parametrized url to the configured Id Provider.\n\n# Errors\n\nThis call fails if Open ID Connect is disabled, misconfigured or the Id Provider is unreachable.\n",
        "operationId": "oidc_init",
        "responses": {
          "200": {
            "description": "",
            "content": {
              "application/json": {
                "schema": {
                  "$ref": "#/components/schemas/AuthCodeRequestURL"
                },
                "example": {
                  "url": "http://someissuer.com/authorize?client_id=someclient&redirect_uri=someuri&response_type=code&scope=somescope&state=somestate&nonce=somenonce&codechallenge=somechallenge&code_challenge_method=S256"
                }
              }
            }
          }
        }
      }
    },
```

File: .generation/post-process/typescript.py (hunk 1 above)
Comment (ChristianBeilschmidt) [reply .generation/post-process/typescript.py:1]: [f4fdd3e](https://github.com/geo-engine/openapi-client/pull/1/commits/f4fdd3e377d9ca286d52e0ea72e931077486cf53)

**Pull request merged**